assets/src/images/maestro.svg

```diff
@@ -0,0 +1,27 @@
+<svg width="750" height="471" viewBox="0 0 750 471" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 48.2 (47327) - http://www.bohemiancoding.com/sketch -->
+    <desc>Created with Sketch.</desc>
```

Comment:
@Sidsector9 are these SVGs optimized for the web? I don't think we want "Created with Sketch" and the html comments. As well, they can be minified from what I've seen others doing.